pushimages.sh

```diff
@@ -53,8 +53,12 @@ else
 fi
 tag=latest
 for image in $imagelist; do
-   echo docker tag kappnav-$image:$tag $org/kappnav-$image:$tag
-   docker tag kappnav-$image:$tag $org/kappnav-$image:$tag
-   echo docker push $org/kappnav-$image:$tag
-   docker push $org/kappnav-$image:$tag
+	# special case for inv images as the image name and repo name are different
+	if [ x$image == x"inventory" ]; then
```

Comment:
This is an image list so the value should never be "inventory".

Reply:
Do you want me to handle the special case in the option value when get pass from setupKAppNavLib.sh?

Reply:
After the meeting, we decide to handle the special case in the setup script.